packages/hooks-react/src/useProfiles.ts

```diff
@@ -1,9 +1,9 @@
-import type { ProfilesData } from '@inplayer-org/inplayer.js';
 import { useMutation, useQuery, type UseMutationOptions, type UseQueryOptions } from 'react-query';
 import { useTranslation } from 'react-i18next';
 import type { GenericFormErrors } from '@jwp/ott-common/types/form';
 import type { CommonAccountResponse } from '@jwp/ott-common/types/account';
 import type { ListProfilesResponse, ProfileDetailsPayload, ProfileFormSubmitError, ProfileFormValues, ProfilePayload } from '@jwp/ott-common/types/profiles';
+import type { ProfilesData } from '@jwp/ott-common/src/services/integrations/jwp/base/types';
```

Comment:
I am not sure this is the right way to import types. Should we consider storing them in `@jwp/ott-common/types` ?

Reply:
Agreed.

Reply:
In a typings refactor PR (not merged) I have separating the typings completely. I did a example in the Cleeng integration folder with a types folder.

The reason for this is to prevent the usage of integration specific types in the application logic. Services should transform models to application data.

Now we are mixing even more integration (JWP) typings. Perhaps this is a good moment to separate this for the JWP integration already?


- Create `./packages/common/src/services/integrations/jwp/types` directory
- Move inplayer DTO typings used only by services (not exposed) to this directory
- When typings are used outside the integration, we could duplicate them in the `./packages/common/typings` folder at feature level

```mermaid
graph TD
    subgraph JWP[JWP Integration]
        JWPTypes[Types]
        JWPClient[Client]
        JWPService[Service]
        
        direction RL
        JWPTypes --> JWPClient
        JWPClient --> JWPService
    end

    subgraph Cleeng[Cleeng Integration]
        direction LR
        CleengTypes[Types]
        CleengClient[Client]
        CleengService[Service]

        CleengTypes --> CleengClient
        CleengClient --> CleengService
    end

    subgraph Common
        Controller        
        Types --> Controller
    end

    Cleeng -- Service calls --> Common
    JWP -- Service calls --> Common
```

Reply:
I would agree, but we're almost rid of this problem: There are only two instances of integration-specific types being imported and used in the web app. One of which is `RegisterFieldOptions` which I'll simply replace with an inline `Record<string, string>` and avoid importing the type that way. The other one is `ProfilesData`, which is practically not even used anymore because we've disabled `Profiles` recently and everything related to that is dead code now, and is to be removed altogether in another upcoming task.

Reply:
Maybe we could start moving `inplayer.ts` types to `src/services/integrations/jwp/types` directory? I am not sure we should have them in `@jwp/ott-common/types`

In case anything is needed outside of the service, we could reexport them from `@jwp/ott-common/types`. @ChristiaanScheermeijer or would you prefer duplicates?

Reply:
I made duplicates. But I would vote for re-exporting to avoid mismatching.

Reply:
Re-exporting them is also fine, but @mirovladimitrovski already solved it nicely 👍